cmake/toolchains/riscv64-100-thead-gnu.toolchain.cmake

```
@@ -0,0 +1,46 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# NOTE: use T-Head compiler:
+#    git clone https://github.com/T-head-Semi/xuantie-gnu-toolchain.git
+#    ./configure --prefix=/opt/riscv
+#    make linux
+#    -DRISCV_TOOLCHAIN_ROOT=/opt/riscv
+
+# To enable cross-compilation with python (for example, on Ubuntu 22.04):
+# $ echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy main >> riscv64-sources.list
+# $ echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy universe >> riscv64-sources.list
+# $ echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy-updates main >> riscv64-sources.list
+# $ echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy-security main >> riscv64-sources.list
+# $ mv riscv64-sources.list /etc/apt/sources.list.d/
+# $ dpkg --add-architecture riscv64
+# $ apt-get update -o Dir::Etc::sourcelist=/etc/apt/sources.list.d/riscv64-sources.list
+# $ apt-get install -y --no-install-recommends libpython3-dev:riscv64
+# $ ln -s /usr/include/riscv64-linux-gnu/ /usr/include/python3.10/
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR riscv64)
+
+set(RISCV64_THEAD ON)
+set(RISCV64_RVV0p7 ON)
```

These variables are not used but defined just in case to distinguish RVV versions.
```suggestion
set(RISCV64_RVV1p0 ON)
```